block-BHaabh/index.html

```diff
@@ -0,0 +1,139 @@
+              </h3>
+              <h4>Semper suscipit posuere apede
+              </h4>
+            <div class="card">
+              <div class="box">
+              <img class="image" src="assets/pic03.jpg" alt="personinfo">
+              <p class="about">Magna</p>
+              <h4>Cipdum dolor</h4>
+            </div>
+            </div>
+
+            <div class="card">
+              <div class="box">
+              <img class="image" src="assets/pic04.jpg" alt="personinfo">
+              <p class="about">Ipsum<p>
+              <h4>Vestibulum comm</h4>
+            </div>
+            </div>
+
+
+            <div class="card">
+              <div class="box">
+              <img class="image" src="assets/pic05.jpg" alt="personinfo">
+              <p class="about">Tempus</p>
+              <h4>Fusce pellent</h4>
+              </div>
+            </div>
+
+
+            <div class="card">
+              <div class="box">
+              <img class="image" src="assets/pic06.jpg" alt="personinfo">
+              <p class="about">Dolore</p>
+              <h4>Praesent placer</h4>
+            </div>
+          </section>  
+
+          <section>
+          <div container>
+          <p class="heading3">Nunc Dignissim </p>
+          <h4>Aliquam erat volutpat nam dui</h4>
+              <div class="thirdsection ">
+                <img class="highway" src="assets/pic01.jpg" alt="highyway">
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Cumque aliquam quaerat sed beatae deleniti iusto numquam ex libero exercitationem veniam doloribus itaque repellendus tenetur vero officiis blanditiis unde, enim qui Lorem ipsum dolor sit amet consectetur adipisicing elit. Ut sunt ratione, praesentium velit quaerat officia ab perspiciatis tenetur eum repudiandae, numquam harum obcaecati. Adipisci distinctio a alias, corrupti fuga perspiciatis!</p>
+                <button class="btn"> <a class="more" href="#">More</a></button>
+            </div>
+
+            <div class="thirdsection ">
+              <img class="highway" src="assets/pic01.jpg" alt="highyway">
+              <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Cumque aliquam quaerat sed beatae deleniti iusto numquam ex libero exercitationem veniam doloribus itaque repellendus tenetur vero officiis blanditiis unde, enim qui Lorem ipsum dolor sit amet consectetur adipisicing elit. Ut sunt ratione, praesentium velit quaerat officia ab perspiciatis tenetur eum repudiandae, numquam harum obcaecati. Adipisci distinctio a alias, corrupti fuga perspiciatis!</p>
+              <button class="btn"> <a class="more" href="#">More</a></button>
+          </div>
+          </div>
+          </section>
+        </main>
+          <hr>
+          <section>
+           <footer class="footer">
+           <div class= "container clearfix">
+         
+            <a href="#"> <h2 class="logo"> © Untitled. Design: TEMPLATED. Images: Unsplash.
```

Comment:
This is just copyright information, not a navigation link, so not required wrap in an anchor tag also use a small tag for copyright information.